Cloudinary/Operation/01_ManageCloudinaryFiles.md

```diff
@@ -0,0 +1,182 @@
+
+3. Click the [LIST UNASSIGNED ITEMS] button in the Cloudinary image attribute box to select the image(s) to be imported. When importing a video from Cloudinary, use the button in the corresponding Cloudinary video attribute box instead of the Cloudinary image attribute box.        
+  The *Unassigned items* window is displayed, listing all unassigned images in the specified Cloudinary folder.
+
+  ![Unassigned items](../../Assets/Screenshots/PIM/Products/List/Attributes/CloudinaryUnassignedItems.png "[Unassigned items]")
+
+  > [Info] Note that only those images located in the folder specified in the Cloudinary attribute configuration are available. If an image is missing, check if the image is included in the specified folder and matches the regular expression if one has been defined. You can use the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list of unassigned items.   
+
+4. Select the checkboxes of all images to be imported from Cloudinary in the list of unassigned items.   
+  The editing toolbar is displayed above the list.
+
+5. Click the [ADD ITEM(S)] button in the toolbar.   
+  The selected images have been imported from Cloudinary and removed from the list of unassigned items.
+
+6. Click the [CANCEL] button in the bottom right corner.   
+  The *Unassigned items* window is closed. The *Edit product* view is displayed again. A preview tile of each selected image is displayed in the Cloudinary image attribute box.   
+
+  ![Cloudinary images](../../Assets/Screenshots/PIM/Products/List/Attributes/CloudinaryImages.png "[Cloudinary images]")
+
+7. Click the preview tile in the attribute image box to edit the image data. The image data fields below
+are displayed in the box side bar:
+  - *Tags*   
+    Enter tag names for the image. Use commas to separate the tag names. A single tag is created for each tag name.
+  - *Description (alt)*   
+    Enter a description for the image which is indicated when the image cannot be displayed. This field is automatically created for each image.
+  - *Title (caption)*   
+    Enter a title for the image. This field is automatically created for each image.
+  - *Filename*   
+    The filename is automatically adopted from the uploaded or imported image. This field is read-only. To edit a filename after uploading, open the corresponding image in Cloudinary.
+  - *Contextual meta data fields*   
+    Enter an appropriate value in the contextual meta data fields specified in the Cloudinary attribute configuration.
+  - *Structured meta data fields*   
+    Enter an appropriate value in the structured meta data fields enabled in the Cloudinary attribute configuration.
+
+    > [Info] Currently only structured meta data fields of the text and the date field type are supported by Actindo. Note to enter a date value in the appropriate date format (yyyy-MM-dd).
```

Comment:
Unsicher, ob die Formulierung "Note to enter" üblich/korrekt ist. Vielleicht:

Note that the valid date format here/in this field is ...
(Näher an unsere Standardformulierung)